commerce-product-api/src/main/java/com/liferay/commerce/product/util/CPCompareUtil.java

```diff
@@ -57,7 +61,18 @@ public static void addCompareProduct(
 			return new ArrayList<>();
 		}
 
-		return cpDefinitionIds;
```

Comment:
@alecsloan Instead of doing this you can use the CPDefinitionHelper getCPCatalogEntry, that already does all the checks

Reply:
Since CPCompareUtil is a static context how are we supposed to use CPDefinitionHelper (not static) inside of the util?

Reply:
we do have to transfor CPCompareUtil in a component than, but let's avoid duplicate logic